dc-model-feign/README.md

```diff
@@ -0,0 +1,6 @@
+# DigitalCollections: Model (Feign)
+
+This library provides classes for handling model object transport using REST-library [OpenFeign](https://github.com/OpenFeign/feign).
+
+- `EndpointErrorDecoder` allows developers to decode HTTP error codes into application-specific ones.
```

Comment:
Aber das sind doch garkeine Application-specific exceptions sondern einfach nur die Standard HTTP-Fehler-Statuscodes auf Exceptions gemappt? Vllt eher "allows developers to handle HTTP error codes as Java checked exceptions." oder sowas.

Reply:
done